addon/components/lt-body.js

```diff
@@ -239,6 +290,55 @@ export default Component.extend({
     this.set('useVirtualScrollbar', fixedHeader || fixedFooter);
   },
 
+  didUpdateAttrs({ newAttrs, oldAttrs }) {
+    this._super(...arguments);
+
+    let newScrollTo = get(newAttrs, 'scrollTo.value');
+    let oldScrollTo = get(oldAttrs, 'scrollTo.value');
+    let newScrollToRow = get(newAttrs, 'scrollToRow.value');
+    let oldScrollToRow = get(oldAttrs, 'scrollToRow.value');
+
+    let targetScrollOffset = 0;
+    let hasReachedTargetScrollOffset = true;
+
+    if (oldScrollTo !== newScrollTo) {
+      targetScrollOffset = newScrollTo;
+      if (newScrollTo > 0) {
+        hasReachedTargetScrollOffset = false;
+      }
+    } else if (oldScrollToRow !== newScrollToRow) {
+      if (newScrollToRow instanceof Row) {
+        let rowElement = document.getElementById(newScrollToRow.get('rowId'));
+        if (rowElement instanceof Element) {
+          targetScrollOffset = rowElement.offsetTop;
+        }
+      }
+    }
+
+    this.set('targetScrollOffset', targetScrollOffset);
+    this.set('hasReachedTargetScrollOffset', hasReachedTargetScrollOffset);
+  },
+
+  rowObserver: observer('rows.[]', function() {
+    run.scheduleOnce('afterRender', this, this.checkTargetScrollOffset);
```

Comment:
This wouldn't account for the timer in line 334. But tbh, I doubt that the timer in 334 could ever cause problems.

---

`run.cancel(...)` inside `_scheduleCheckTargetScrollOffset()` is unnecessary.

Relevant code in Backburner:
https://github.com/ebryn/backburner.js/blob/master/lib/backburner/deferred-action-queues.js#L38-L42
https://github.com/ebryn/backburner.js/blob/master/lib/backburner/queue.js#L33-L37